### PR TITLE
Remove redundant lines

### PR DESCRIPTION
**Pull request by Youssef1313**
- `System` is imported by default in visual basic. so I removed `Imports System`
- calling `Flush` isn't necessary, the using statement should take care of that.
- Used `var`.
- Followed comments convention.

### CODE REVIEW

File: snippets/csharp/VS_Snippets_CLR/FInfo Class/CS/finfo class.cs [hunk 1]
```diff
@@ -31,23 +31,23 @@ public static void Main()
...
 
-            //Ensure that the target does not exist.
+            // Ensure that the target does not exist.
             fi2.Delete();
 
-            //Copy the file.
+            // Copy the file.
             fi1.CopyTo(path2);
-            Console.WriteLine("{0} was copied to {1}.", path, path2);
+            Console.WriteLine($"{path1} was copied to {path2}.");
```
Comment (BillWagner) [inline snippets/csharp/VS_Snippets_CLR/FInfo Class/CS/finfo class.cs]: There;s a typo in your change:

```suggestion
            Console.WriteLine($"{path} was copied to {path2}.");
```

**Comment by BillWagner** [comment]
@Youssef1313 This now has some merge conflicts. Once those are resolved, we can :shipit:.

Thanks for making these updates.

**Comment by Youssef1313** [comment]
@BillWagner, Well, my fork was deleted. Not sure if there is a good way to resolve that conflicts ? or I should open a new PR ?

**Comment by BillWagner** [comment]
@Youssef1313 You may see if you can recover the fork from GitHub. Otherwise, yes, a new PR would be cleanest.

**Pull request closed without merging**